### ISSUE TITLE
SQLite Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [FreezeWarp/freeze-messenger#72]
```
Support PHP's SQLite3 Driver

http://www.php.net/manual/en/book.sqlite3.php
```

Original issue reported on code.google.com by `JosephTP...@gmail.com` on 13 Jul 2011 at 8:54
- Blocked on: #69
- Merged into: #71
